src/storage/byte_order.c

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * byte_order.c - Definitions related to disk storage order
```

Comment:
CUBRID is currently supporting only LINUX and Windows. <arpa/inet.h> and <winsock2.h> implement the byte ordering functions in Linux and Windows, respectively. And CUBRID's target OS seems not to be changed in the future. Why do we need these declarations and implementations for unsupported OSs?

For reference:
```
#if defined(HPUX) || defined(_AIX) || defined(sparc)
#define OR_BYTE_ORDER OR_BIG_ENDIAN
#else /* HPUX || _AIX || sparc */
#define OR_BYTE_ORDER OR_LITTLE_ENDIAN	/* WINDOWS, LINUX, x86_SOLARIS */
#endif /* HPUX || _AIX || sparc */

#if defined(HPUX) || defined(_AIX) || defined(WINDOWS) || defined(LINUX)
#define OR_HAVE_NTOHS
#define OR_HAVE_NTOHL
#define OR_HAVE_HTONS
#define OR_HAVE_HTONL
#endif /* HPUX || _AIX || WINDOWS || LINUX */
```

Reply:
It was moved as-is to minimize the impact of code changes.

I think removing code for other OS should be another issue.
I will create a separate issue for this and share it.

Reply:
I created the issue below.
http://jira.cubrid.org/browse/CBRD-24646